source/base/logstream.cc

```diff
@@ -385,6 +385,7 @@ LogStream::get_prefixes() const
   bool                     exists         = false;
   std::stack<std::string> &local_prefixes = prefixes.get(exists);
 
+#  if 0 // FIXME
```

Comment:
What had you planned to do here?

Reply:
@bangerth I will make logstream a friend and grab the corresponding information directly out of the map.